anta/custom_types.py

```diff
@@ -85,6 +85,12 @@ def bgp_multiprotocol_capabilities_abbreviations(value: str) -> str:
     BeforeValidator(interface_autocomplete),
     BeforeValidator(interface_case_sensitivity),
 ]
+SpeedInterface = Annotated[
```

Comment:
Can we name that something else? Ethernet_Interface maybe?

Reply:
EternetInterface